packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.test.js

```diff
@@ -118,6 +119,23 @@ describe('ListToVirtualizedList', () => {
 		});
 	});
 
+	it('should support custom cell renderer', () => {
+		const clonedProps = cloneDeep(props);
```

Comment:
You don't need to deep clone the props, just spread the props, and then add your `cellDictionary` props in your component.
```
<ListToVirtualizedList {...props} cellDictionnary={customDictionnary} />
```